nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/WebClient.java

```diff
@@ -67,12 +70,14 @@ static <T> T builder(Protocol<T> protocol) {
 
         private URI baseUri;
         private Tls tls;
-        private SocketOptions channelOptions;
+        private final SocketOptions.Builder socketOptionsBuilder = SocketOptions.builder();
```

Comment:
The existing field is called `channelOptions`, let's call the new one `channelOptionsBuilder`.